cypress/support/cypress-commands/commands.js

```diff
@@ -699,8 +686,18 @@ Cypress.Commands.add('launchApp', (appType, appCallSign) => {
         },
       },
     };
-    requestMap.params.intent.data = data;
```

Comment:
Remove comments from line 695 and 696. 
Add comment before creating the messageIntent object
// Creating intent and request map to be sent to the app on launch

Reply:
Added comment